assets/css/home.css

```diff
@@ -24,22 +24,24 @@ body {
 
 .catagory {
     align-items: center;
-    background: #fff;
+    background-color: #fff;
     border-bottom: 1px solid #dbdbdb;
-    border-top: 1px solid #dbdbdb;
     display: flex;
     height: 60px;
-    left: 260px;
+    justify-content: space-between;
+    margin-left: 258px;
     overflow-x: auto;
     overflow-y: hidden;
-    padding: 0 20px;
-    position: fixed;
+    padding: 10px 2%;
+    position: sticky;
     top: 60px;
     width: calc(100% - 250);
+    z-index: 10;
 
 }
 
 .catagory_btn {
```

Comment:
Use hyphen `-` instead of underscore `_`

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix